textChangeTest.py

```diff
@@ -0,0 +1,10 @@
+import random
+from memoize.memoized import Memoized
```

Comment:
Assume the package is installed and call it with the short version.

Reply:
wat

Reply:
Install the package and import it using the shorter way used in `example.py` (`from memorized import Memoized`). Then, run the test assuming the package is installed.